wp-resources/plugins/sil-dictionary-webonary/webonary/Webonary_Parts_Of_Speech.php

```diff
@@ -170,9 +170,12 @@ private function GetPartsOfSpeechFromWordpress(): array
 		/** @var WP_Term $item */
 		foreach ($list as $item) {
 
+			if (empty($item->term_id))
+				continue;
+
 			$part = [
 				'abbreviation' => $item->term_id,
-				'lang' => $this->current_lang_code,
+				'lang' => 'en',
```

Comment:
The lang is no longer a variable. Is that on purpose?

Reply:
@johntseng in a MySQL dictionary there is no language associated with the parts of speech, so it doesn't matter at this point.